gengokit/generator/gen.go

```diff
@@ -30,8 +31,12 @@ func GenerateGokit(sd *svcdef.Svcdef, conf gengokit.Config) (map[string]io.Reade
 
 	codeGenFiles := make(map[string]io.Reader)
 
+	// Remove the suffix "-service" since it's added back in by templatePathToActual
+	svcname := strings.TrimSuffix(path.Base(conf.GoPackage), "-service")
```

Comment:
This is better, though actually making a value dedicated to this on `conf` may be nicer. 

It would remove the need for any of these added comments because it would be clear from the code.

Though this is fine too.